apps/website/src/content/docs/combobox.mdx

```diff
@@ -3,13 +3,12 @@ title: ComboBox
 description: A dropdown list that allows users to type a value to filter the options.
 propsPath: '@itwin/itwinui-react/esm/core/ComboBox/ComboBox.d.ts'
 thumbnail: ComboBox
+related:
+  - dropdownmenu
+  - select
+  - popover
```

Comment:
Are we not having `Label` in this related components list any more?

Reply:
we don't have a label page anymore